Entity/SnapshotManager.php

```diff
@@ -133,6 +133,7 @@ public function findEnableSnapshot(array $criteria)
             throw new \RuntimeException('please provide a `pageId`, `url`, `routeName` or `name` as criteria key');
         }
 
+        $query->orderBy('s.publicationDateStart', 'DESC');
```

Comment:
BTW for me it's BC break.

By default, it's ordered by ID. So if a user take the first element considering id order, it would not get the same result after that.